utils/mcp_client.py

```
@@ -0,0 +1,531 @@
+            await asyncio.wait_for(process.wait(), timeout=2)
+        except asyncio.TimeoutError:
+            process.terminate()
+            await process.wait()
+
+    async def _request(
+        self,
+        method: str,
+        params: Mapping[str, Any] | None = None,
+    ) -> dict[str, Any]:
+        await self.start()
+        assert self._process and self._process.stdin and self._process.stdout
```

Ruff (0.15.20)

[warning] Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)